ecpp

```diff
@@ -1,35 +1,75 @@
 #!/usr/bin/python
 
+# Author: Hubert Kario, Stefan Dordevic
+# Released under Gnu GPL v2.1, see LICENSE file for details
+
 from __future__ import print_function
 import ConfigParser
 import sys
 import StringIO
 from verify import *
+from convert import *
```

Comment:
star imports are not recommended, they make it harder for static analysers to detect unused code